pkgs/applications/networking/irc/catgirl/default.nix

```
@@ -0,0 +1,22 @@
+{lib, stdenv, fetchurl, ncurses, libressl, man, pkg-config, ctags}:
+
+stdenv.mkDerivation rec {
+  pname = "catgirl";
+  version = "1.6";
+
+  src = fetchurl {
+    url = "https://git.causal.agency/catgirl/snapshot/${pname}-${version}.tar.gz";
+    sha256 = "0shg02zidqqmvywqqsaazlgg9rd5lhhrvjx6n0lzmdfaawxywciv";
+  };
+
+  nativeBuildInputs = [ pkg-config ncurses libressl man ctags ];
```

```suggestion
  nativeBuildInputs = [ pkg-config man ];
  
  buildInputs = [ ncurses libressl ctags ];
  
  strictDeps = true;
```

Most likely this is correct but I am not completely sure.

---

seems `ctags` still needs to be in the `nativeBuildInputs`